### ISSUE TITLE
OSX/PowershellCore/VSCode extension - any use of functions in a script crashes session

### ISSUE THREAD

**Issue by jskii** [PowerShell/vscode-powershell#1164]
### System Details

- Operating system name and version: OSX 10.13.1 High Sierra
- VS Code version: 1.19.2
- PowerShell extension version: 1.5.1

```
1.19.2
490ef761b76b3f3b3832eff7a588aac891e5fe80
x64

Major  Minor  Build  Revision
-----  -----  -----  --------
1      5      1      0

code --list-extensions --show-versions
ms-vscode.PowerShell@1.5.1

Name                           Value
----                           -----
PSVersion                      6.0.0
PSEdition                      Core
GitCommitId                    v6.0.0
OS                             Darwin 17.2.0 Darwin Kernel Version 17.2.0: Fri Sep 29 18:27:05 PDT 2017; root:xnu-4570.20.62~3/RELEASE_X86_64
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### Issue Description

Any time a function is used in a powershell script that loads into the editor window, the powershell session crashes. This happened in realtime while incorporating images for this example:

1. Write a basic script to write a string. Works fine.
2. Edit that script to feed a string to a function to write on screen. This crashes immediately after adding in the function call. It happens before saving the file, immediately after typing the { curly brace to note the start of the function script block.

### Attached Logs

Verbose debug logs from sessions which crash.

![no_function](https://user-images.githubusercontent.com/3965623/34951121-3735225a-f9e3-11e7-8493-d5e1cbd2fa0b.png)
[vscode-powershell-verbose-logs.zip](https://github.com/PowerShell/vscode-powershell/files/1632236/vscode-powershell-verbose-logs.zip)
![with_function](https://user-images.githubusercontent.com/3965623/34951122-3761dd18-f9e3-11e7-862d-cdd4506c5c7a.png)

**Comment by jskii** [comment]
After looking into the logs, this looks like the same issue as reported here:

https://github.com/PowerShell/vscode-powershell/issues/1017

I will leave this open if you require more information, but this can probably be closed and tracked in the older issue report. Thanks!

**Comment by jskii** [comment]
This appears to have been fixed in version 1.6.0 so I will close this request.
